Application/Models/Role.cs

```diff
@@ -1,11 +1,10 @@
 using Microsoft.AspNetCore.Identity;
+using MongoDB.Bson;
 
 namespace Application.Models;
 
-public class Role : IdentityRole<int>
+public class Role : IdentityRole<ObjectId>
```

Comment:
Если роли остаются в sql, id должен быть int
```suggestion
public class Role : IdentityRole<int>
```